active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,622 @@
+ the Title, Message, or custom content.
+ 
+
+ The accompanying information should aim to communicate the following to use that Severity.
+ - Error: An error or problem that has occurred.
+ - Warning: A condition that might cause a problem in the future.
+ - Success: A long-running and/or background task has completed.
+ - Default: General information that requires the user's attention.
+
+Icons and color should not be the only UI components signifying meaning for your notification. 
+Text in the notification's Title and/or Message should be included to display information.
+
+
+## Message 
+
+Text in your notification will not be a constant length in all languages. For the Title and Message property 
+this may impact whether your notification will expand to a second line. 
+We recommend you avoid positioning based on message length or other UI elements set to a specific language.
+
+The notification will follow standard mirroring behavior when localized to/from languages that are right to left (RTL) or left to right (LTR). The icon will only mirror if there is directionality.
+
+Please view the guidance for 
+[Adjust layout and fonts, and support RTL](https://docs.microsoft.com/en-us/windows/uwp/design/globalizing/adjust-layout-and-fonts--and-support-rtl) 
+for more information about text localization in your notification.
+
+
+# API Notes
+
+### Notable Properties  
+| Name | Description |
+|:-:|:--|
+| IsOpen| Gets or sets a value that determines the visibility of the InfoBar. By default, is set to false. |
+| Severity | Gets or sets a value that indicates the  color and icon to style the InfoBar. It will also define the assistive technology settings. |
+| IsUserDismissable| Gets or sets a boolean that indicates whether the user will be able to dismiss the InfoBar.
+
+
+### Events  
+| Name | Description |
+|:-:|:--|
+| CloseButtonClick | Occurs after the close button has been clicked. |
+| Closed | Occurs after the info bar is closed. |
+| Closing |Occurs just before the info bar begins to close. |
+
+
+# API Details
+
+```c++
+enum InfoBarCloseReason
+{
+    CloseButton = 0,
+    Programmatic = 1,
+};
+
+enum InfoBarSeverity
+{
+    Error = 0,
+    Warning = 1,
+    Success = 2,
+    Default = 3,
+}
+
+runtimeclass InfoBarClosedEventArgs
+{
+    InfoBarCloseReason Reason{ get; };
+};
+
+runtimeclass InfoBarClosingEventArgs
+{
+    InfoBarCloseReason Reason{ get; };
+    Boolean Cancel;
+    Windows.Foundation.Deferral GetDeferral();
+};
+
+unsealed runtimeclass InfoBarTemplateSettings : Windows.UI.Xaml.DependencyObject
+{
+    InfoBarTemplateSettings();
+    Windows.UI.Xaml.Controls.IconElement IconElement;
+
+    static Windows.UI.Xaml.DependencyProperty IconElementProperty{ get; };
+}
+
+unsealed runtimeclass InfoBar : Windows.UI.Xaml.Controls.ContentControl
+{
+    InfoBar();
+
+    String Title;
+    String Message;
+
+    Boolean IsOpen;
+    Boolean IsUserDismissable;
```

Comment:
Should this be `IsCloseButtonVisible`?

Reply:
@mikeh brought this up earlier in the [conversation](https://github.com/microsoft/microsoft-ui-xaml-specs/pull/94#discussion_r487346017):
> "Close" is a more standard term for this action than "dismiss". And this should follow the "Can" pattern for consistency. So "CanClose" rather than "IsUserDismissable". (For example similar to [TextBox.CanUndo](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.TextBox.CanUndo).)

We avoided IsCloseButtonVisible to remain flexible with devices in the future (like touch-enabled) where a close button may not be visible but the user is still able to close the bar via a swipe or other gesture. What are your thoughts on CanClose?

Reply:
Guessing you mis-tagged and this was meant for @MikeHillberg :-)

Reply:
Sure, `CanClose` is good too.